UI/cmake/ResourceFiles.cmake

```diff
@@ -50,6 +49,9 @@ set(128x128_ICONS
     app-browser.png
     app-browser-dark.png
 )
+set(svg_ICONS
```

Comment:
Just declaring the `svg_ICONS` variable on its own does not do anything. See how `48x48_ICONS` is used below.